applications/projects/runSofa/Main.cpp

```diff
@@ -544,5 +544,6 @@ int main(int argc, char** argv)
 
     sofa::simulation::common::cleanup();
     sofa::simulation::graph::cleanup();
+    PluginManager::getInstance().cleanup();
```

Comment:
I think it would be better to set this line directly into the cleanup of sofa::helper()
ie. here: https://github.com/sofa-framework/sofa/blob/9fda8923672664c1e98060400ed066c4c7735d1c/Sofa/framework/Helper/src/sofa/helper/init.cpp#L49

which is called "cascadingly" by the other cleanups()

Reply:
ok, you are right, better than polluting the main...